src/lib/messenger.ts

```diff
@@ -49,7 +49,8 @@ type StringMessage = StandardMessage<
 	| 'get-page-url'
 	| 'passback-refresh'
 	| 'viewport'
-	| 'scroll',
+	| 'scroll'
+	| 'click',
```

Comment:
I think this one I discussed with @Jakeii being necessary to track click throughs, as we have a couple or research and tracking props in the Fabric video component.
I assumed it would be as a result of
`const postAndListen = (arg: Message)` where the `Message` type could be a click from the window?
But Im open to be corrected